dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/dispatch/host/LowerWeightHostManager.java

```diff
@@ -78,7 +78,7 @@ public void init() {
         this.workerHostWeightsMap = new ConcurrentHashMap<>();
         this.lock = new ReentrantLock();
         this.executorService = Executors.newSingleThreadScheduledExecutor(new NamedThreadFactory("LowerWeightHostManagerExecutor"));
-        this.executorService.scheduleWithFixedDelay(new RefreshResourceTask(), 0, 5, TimeUnit.SECONDS);
+        this.executorService.scheduleWithFixedDelay(new RefreshResourceTask(), 0, 1, TimeUnit.SECONDS);
```

Comment:
Can we add listener to `ServerNodeManager` and register `RefreshResourceTask` as a listener so that we can immediately refresh the resources when `ServerNodeManager` updates the nodes?

Reply:
Good idea. I found that `WorkerDataListener` just handle add and remove event, but not update event.
I will try to add update event handle for it.